### PR TITLE
First gapic bigtable gen

### PR DESCRIPTION
**Pull request by andreamlin**
Copied over from api-client-staging/master. Directory structure changed to match google-cloud-java convention.

### CODE REVIEW

**Comment by andreamlin** [comment]
PTAL

File: google-cloud-bigtable/pom.xml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      <groupId>${project.groupId}</groupId>
+      <artifactId>google-cloud-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>google-cloud-core-grpc</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>proto-google-cloud-bigtable-v2</artifactId>
```
Comment (garrettjonesgoogle) [inline google-cloud-bigtable/pom.xml]: Why don't you need proto-google-cloud-bigtable-admin-v2 also?

**Comment by garrettjonesgoogle** [comment]
You need to update the room pom to include this in the build.

File: google-cloud-bigtable/pom.xml (hunk 1 above)
Comment (andreamlin) [reply google-cloud-bigtable/pom.xml]: Correct, and also added grpc bigtable-admin dependency.

**Comment by andreamlin** [comment]
Added the requisite changes to the pom.xmls.
PTAL

File: google-cloud-bigtable/pom.xml [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-cloud-bigtable-v2</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>proto-google-cloud-bigtable-admin-v2</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-cloud-bigtable-admin-v2</artifactId>
```
[the remark this answers is not in the document]
Comment (andreamlin) [reply google-cloud-bigtable/pom.xml]: Done.

File: google-cloud-bigtable/pom.xml [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.truth</groupId>
+      <artifactId>truth</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-iam-v1</artifactId>
```
[the remark this answers is not in the document]
Comment (andreamlin) [reply google-cloud-bigtable/pom.xml]: Done.

**Comment by andreamlin** [comment]
Fixed up the pom.xml. PTAL

**Pull request merged**